provider/openstack/local_test.go

```diff
@@ -513,7 +513,7 @@ func (s *localServerSuite) TestStartInstanceHardwareCharacteristics(c *gc.C) {
 	env := s.ensureAMDImages(c)
 	err := bootstrapEnv(c, env)
 	c.Assert(err, jc.ErrorIsNil)
-	_, hc := testing.AssertStartInstanceWithConstraints(c, env, s.callCtx, s.ControllerUUID, "100", constraints.MustParse("mem=1024"))
+	_, hc := testing.AssertStartInstanceWithConstraints(c, env, s.callCtx, s.ControllerUUID, "100", constraints.MustParse("mem=1024 arch=amd64"))
```

Comment:
I wonder why this just flagged up now? I would have thought we would have defaulted to amd64?

Reply:
https://github.com/juju/juju/blob/2.9/environs/bootstrap/bootstrap.go#L433-L445

It's been there for quite a while, hence my question about how this test could have been working before on non-amd64 boxes.